integration/rpctest/rpc_harness_test.go

```diff
@@ -351,7 +351,7 @@ func testGenerateAndSubmitBlock(r *Harness, t *testing.T) {
 
 	// Now generate a block with the default block version, and a zero'd
 	// out time.
-	block, err := r.GenerateAndSubmitBlock(txns, -1, time.Time{})
+	block, err := r.GenerateAndSubmitBlock(txns, nil, -1, time.Time{})
```

Comment:
at the level of block submitting there is is no verification so nil is enough here. When `submitBlock` endpoint will receive this block it will correctly deserialize it to empty slice (not nil)